CI/travis.linux.after_success.sh

```diff
@@ -5,7 +5,8 @@ set -e
 # we deploy only certain builds
 if [ "${DEPLOY}" = "deploy" ]; then
 
-  if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+  if [ "${TRAVIS_EVENT_TYPE}" == "cron" ] \
+    && [ "${CC}" = "gcc" ] && [ "${Q_OR_C_MAKE}" = "qmake" ]; then
```

Comment:
This is a bit inconsistent. In the first comparison you compare with `==`, but then you compare with `=`.

Reply:
Updated.